Yona/Yona/AppDelegate.swift

```diff
@@ -16,14 +16,14 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
         
-        var rootController : UINavigationController
-
-        rootController = getScreenNameToDisplay()
-        
-        if let window = self.window {
-            window.rootViewController = rootController
-        }
- 
+//        var rootController : UINavigationController
+//
+//        rootController = getScreenNameToDisplay()
+//        
+//        if let window = self.window {
+//            window.rootViewController = rootController
+//        }
+// 
```

Comment:
I intentionally put this code in comment, in order to make my work fast. It is a required code. Please don't remove this code.